server/src/main/resources/db/2020.06.03.16.00.00_createInitialDb.sql

```diff
@@ -3,12 +3,20 @@ CREATE TABLE memberProfile(
    uuid varchar PRIMARY KEY,
    name varchar,
    role varchar,
-   pdlId bigint,
+   pdlId varchar,
    location varchar,
    workEmail varchar,
    insperityId varchar,
    startDate date,
    bioText varchar
 );
 
-
+drop table if exists checkins;
+CREATE TABLE checkins(
+   id varchar PRIMARY KEY,
+   teamMemberId varchar,
```

Comment:
are we not marking this as foreign key with this PR ?

Reply:
Good catch. Updated PR.